NDB.Covid19/NDB.Covid19/Locales/nn.json

```diff
@@ -27,18 +27,19 @@
   "WELCOME_PAGE_THREE_BODY_ONE": "Mobilen din lagar ein tilfeldig ID-nøkkel kvart 15. minutt. Når to mobilar er i nærleiken av kvarandre, sender dei desse nøklane til kvarandre.",
   "WELCOME_PAGE_THREE_BODY_TWO": "Du kan dele informasjonen med andre dersom du får covid-19. Da får andre beskjed i appen dersom dei har vore i nærleiken av deg.",
   "WELCOME_PAGE_THREE_INFOBOX_BODY": "Appen nyttar Bluetooth. Teknologien bak appen er utvikla av Apple og Google. ",
-  "WELCOME_PAGE_ACCESSIBILITY_ANNOUNCEMENT_PAGE_CHANGED_TO_THREE": "Side 3 av 4",
+  "WELCOME_PAGE_ACCESSIBILITY_ANNOUNCEMENT_PAGE_CHANGED_TO_THREE": "Side 4 av 4",
   "WELCOME_PAGE_FOUR_TITLE": "Appen passar på personopplysningane dine",
   "WELCOME_PAGE_FOUR_BODY_ONE": "Du kan ikkje sjå kven eller kor mange du har vore i nærleiken av, og dei kan ikkje sjå deg.",
   "WELCOME_PAGE_FOUR_BODY_TWO": "Dersom du varslar om at du er smitta, kan ikkje andre sjå at det er du som gjer det.",
   "WELCOME_PAGE_FOUR_BODY_THREE": "Appen registrerer ikkje data om kvar du er.",
-  "WELCOME_PAGE_ACCESSIBILITY_ANNOUNCEMENT_PAGE_CHANGED_TO_FOUR": "Side 4 av 4",
+  "WELCOME_PAGE_ACCESSIBILITY_ANNOUNCEMENT_PAGE_CHANGED_TO_FOUR": "Side 3 av 4",
```

Comment:
this field name does not have sense if it is page 3

Reply:
It's a public input we got a long time ago. 
I think it's because page 3 and 4 have been change between Danish and Norwegian